includes/helper.php

```diff
@@ -98,11 +98,23 @@ function fiad_check_db_error_file(){
 	}
 }
 
-if(!function_exists('fiad_get_maintenance_mode_option')){
-	function fiad_get_maintenance_mode_option($key){
-		return fiad_get_option($key, get_option('fiad_maintenance_mode'));
+if(!function_exists('fiad_get_csm_mode_option')){
+	function fiad_get_csm_mode_option($key){
+		return fiad_get_option($key, get_option('fiad_csm_mode'));
 	}
 }
+if(!function_exists('fiad_check_maintenance_mode_file')){
+	function fiad_check_maintenance_mode_file(){
+		return (file_exists(WP_CONTENT_DIR . '/templates/coming-soon.php'));
+	}
+}
+
+if(!function_exists('fiad_check_coming_soon_file')){

```

Comment:
Seems duplicate with `fiad_check_maintenance_mode_file`, could we use one function with param?